x-pack/plugins/alerting/common/alert_schema/field_maps/alert_field_map.ts

```diff
@@ -142,6 +149,11 @@ export const alertFieldMap = {
     array: false,
     required: false,
   },
+  [ALERT_FLAPPING_HISTORY]: {
```

Comment:
TODO: should we add a field for pending_recovered_count? This is a new field in the Alert meta used for flapping